broker-core/src/main/java/io/zeebe/broker/subscription/message/processor/MessageEventProcessors.java

```diff
@@ -77,7 +77,8 @@ public static void addMessageProcessors(
         .onCommand(
             ValueType.MESSAGE_SUBSCRIPTION,
             MessageSubscriptionIntent.RESET,
-            new ResetMessageCorrelationProcessor(messageState, subscriptionState))
+            new ResetMessageCorrelationProcessor(
+                messageState, subscriptionState, subscriptionCommandSender))
```

Comment:
We should also change the intent to "REJECT" to be aligned with the subscription command.